src/api/api.js

```diff
@@ -1,10 +1,14 @@
 import axios from "axios";
 import { API_URL } from "./config";
 
-export const register = (email, password) => {
-  return axios.post(`${API_URL}/auth/register`, { email, password });
+// export const register = (email, phone, password) => {
```

Comment:
Please remove the code if you don't use it. Comment code is a bad habit. We can always use git to go back to previous change

Reply:
Done